### PR TITLE
Global reward balance

### PR DESCRIPTION
**Pull request by iamkahvi**
Adding global variable to widget

### CODE REVIEW

File: packages/vanilla-components/src/components/stats-container/stats-container.tsx [hunk 1]
```diff
@@ -87,12 +88,12 @@ export class StatsContainer {
...
 
   getRewardBalance(statVariables) {
-    const { type, unit, valuetype } = statVariables;
+    const { type, unit, valuetype, global } = statVariables;
     // passing in CASH/USD as CASH_USD into this webcomponent so pathToRegexp doesn't split it and it stays all in unit.
     // this converts the _ back to / so that it matches the unit from the backend
     // possibly change the unit in the backend to avoid this?
     const newUnit = unit.replace(/_/, "/")
-    const rewardBalance = this.stats['rewardBalances'].find(rb => rb.type === type && rb.unit === newUnit);
+    const rewardBalance = this.stats[global === "global" ? 'rewardBalanceGlobal' : 'rewardBalance'].find(rb => rb.type === type && rb.unit === newUnit);
```
Comment (jorgecgll) [inline packages/vanilla-components/src/components/stats-container/stats-container.tsx]:
```suggestion
    const rewardBalance = this.stats[global === "global" ? "rewardBalanceGlobal" : "rewardBalance"].find(rb => rb.type === type && rb.unit === newUnit);
```

Or , you could probably make an assumption that if `global` exists, then it's global.

```suggestion
    const rewardBalance = this.stats[`rewardBalance${global && "Global"} `].find(rb => rb.type === type && rb.unit === newUnit);
```

**Comment by iamkahvi** [comment]
Doing some testing

File: packages/vanilla-components/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@saasquatch/vanilla-components",
-  "version": "1.0.8",
+  "name": "@kahvi.patel/vanilla-components",
```
Comment (jorgecgll) [inline packages/vanilla-components/package.json]:
```suggestion
  "name": "@saasquatch/vanilla-components",
```

File: packages/vanilla-components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.0.1",
```
Comment (jorgecgll) [inline packages/vanilla-components/package.json]:
```suggestion
  "version": "1.0.8",
```

**Pull request merged**